build-system/routes/list.js

```diff
@@ -74,6 +73,16 @@ const squareImgUrl = width => {
   return `http://picsum.photos/${width}?${randInt(50)}`;
 };
 
+const randomFalsy = () => {
+  const rand = Math.floor(Math.random() * Math.floor(3));
```

Comment:
`Math.floor(3)` is just `3`.

Reply:
Ohhhhhh I know what this is doing now. I floored the wrong variable lol.

Reply:
Fixed this.